src/rock/models/finances/models/Transaction.js

```diff
@@ -168,6 +178,7 @@ export default class Transaction extends Rock {
         order: [["TransactionDateTime", "DESC"]],
         where: TransactionDateTime ? [{ TransactionDateTime }] : null,
         include: [
+          { model: TransactionDetail.model, where: { AccountId: { $in: deductibleAccounts } } },
```

Comment:
@jbaxleyiii this is the only one I'm kinda unsure about, since it has multiple includes